internal/config/authentication.go

```diff
@@ -379,6 +379,7 @@ type AuthenticationMethodOIDCProvider struct {
 	ClientSecret    string   `json:"clientSecret,omitempty" mapstructure:"client_secret"`
 	RedirectAddress string   `json:"redirectAddress,omitempty" mapstructure:"redirect_address"`
 	Scopes          []string `json:"scopes,omitempty" mapstructure:"scopes"`
+	PKCE            bool     `json:"pkce,omitempty" mapstructure:"pkce"`
```

Comment:
how about `use_pkce`? sort of matches some of our existing boolean configs that way

Reply:
@markphelps That sounds good to me.